doc/user/content/install.md

```diff
@@ -19,6 +19,15 @@ see the [Versions page](/versions).
 
 **Have any questions?** [Contact us](https://materialize.io/contact/)
 
+## Docker
+
+We provide the `materialize/materialized` image in docker hub, if you already have docker
+you can run materialized with your tools in the usual way. For example:
```

Comment:
Content wise this looks good to me, but grammar wise Docker Hub and Docker should be capitalized. Also the sentence is a run on.